tests/Functional/ParagraphsTest.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_theme\Functional;
+
+use Drupal\Tests\BrowserTestBase;
+
+/**
+ * Tests that theme configuration is correctly applied.
+ */
+class ParagraphsTest extends BrowserTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static $modules = [
+    'config',
+    'system',
+    'node',
+    'oe_theme_helper',
+    'oe_paragraphs_demo',
+    'oe_content_timeline_field',
+  ];
+
+  /**
+   * Test Accordion item paragraph form.
+   */
+  public function testAccordionParagraph(): void {
```

Comment:
The description says we test the item, but it's not reflected in the function name.